src/validator.ts

```diff
@@ -53,7 +53,7 @@ const matchingSignatures =
   (list: ReadonlyArray<ClassMember<Linked>>, member: Method<Linked>) =>
     list.some(m => m.kind === 'Method' && m.name === member.name && canBeCalledWithArgs(m, member))
 
-const isNotEmpty = (node: notEmpty) => node.body!.sentences.length !== 0
+const isNotEmpty = (node: notEmpty) => node.body?.sentences.length !== 0
```

Comment:
This is a bit weird... If I don't understand it wrong this should not type, since `sentences` might be null and not have a `length`. Shouldn't this be `node.body?.sentences?.length` ?

Reply:
me neither. I was trying to avoid a `body` reference undefined.